templates/article.html

```diff
@@ -130,8 +131,21 @@ <h2 class="card-title">{{ article.title }}</h2><!--title-->
           <textarea class="form-control" rows="3" name="body" placeholder="write something here..." required="required"></textarea>
         </div>
 
-        <button class="btn btn-primary" type="submit" style="float:right">comment</button>
+        <button id="btn" class="btn btn-primary" type="submit" style="float:right"  onClick="prevent()">comment</button>
```

Comment:
@1500h 

Why do you have the` prevent()` method, which is not defined anywhere?

-Hui